src/DependenciesPackager/CrossgenContext.cs

```diff
@@ -192,22 +365,25 @@ private DirectoryInfo CreateSubDirectory(string cacheBasePath, PackageEntry pack
                 package.Library.Identity.Version.ToNormalizedString().ToLowerInvariant(),
                 Path.GetDirectoryName(asset.RelativePath));
 
-            _logger.LogInformation($"Creating sub directory on {subDirectoryPath}.");
-
             return Directory.CreateDirectory(subDirectoryPath);
         }
 
-        private string GetTempStagingFolder(string parent)
+        public void CreateResponseFile()
         {
-            var publishFolder = _context.GetStagingFolderPath(parent);
-
-            if (!Directory.Exists(publishFolder))
-            {
-                _logger.LogInformation($"Creating directory {publishFolder}.");
-                Directory.CreateDirectory(publishFolder);
-            }
+            _responseFilePath = Path.Combine(_destinationFolder, "aspnet-crossgen.rsp");
+            var builder = new StringBuilder();
+            builder.AppendLine($"/Platform_Assemblies_Paths \"{string.Join(";", _referenceAssemblyPaths)}\"");
```

Comment:
IIRC a response file usually requires one line per argument (at least according to System.CommandLine), but each tool handles rsp slightly different